app/_src/gateway/upgrade/backup-and-restore.md

```diff
@@ -0,0 +1,270 @@
+---
+title: Backup and restore
+content_type: how-to
+purpose: Learn how to back up and restore Kong Gateway
+---
+
+Before you start any upgrade, back up your {{site.base_gateway}} data. 
+Kong supports two back up methods for {{site.base_gateway}} entities: database backup and declarative backup.
```

Comment:
Please ensure to rename "database backup"  to "database-native backup" on this page. Here is an example:

```suggestion
Kong supports two back up methods for {{site.base_gateway}} entities: [database-native backup](#database-native-backup) and [declarative backup](#declarative-backup).
```